packages/recomponents/src/components/r-radio/r-radio.spec.js

```diff
@@ -2,8 +2,6 @@ import {mount, shallowMount} from '@vue/test-utils';
 import {renderToString} from '@vue/server-test-utils';
 import RRadio from './r-radio.vue';
 
-const $t = () => 'custom label';
```

Comment:
What's that?

Reply:
This was causing a linting error for an unused const :)

Reply:
old mock for i18n lib